migrator.sh

```diff
@@ -293,10 +301,38 @@ pull_images_from_v1() {
   for i in ${FULL_IMAGE_LIST}
   do
     push_pull_image "pull" "${V1_REGISTRY}/${i}"
+    retag_image "${V1_REGISTRY}/${i}" "${V2_REGISTRY}/${i}"
+    push_pull_image "push" "${V2_REGISTRY}/${i}"
   done
   echo -e "${OK} Successully pulled all images from ${V1_REGISTRY} to your local system"
 }
 
+pull_retag_push_remove_image() {
+  GLOBIMAGE="${1}"
+
+  echo -e "\n${INFO} Processing ${GLOBIMAGE}"
+
+  IMAGE_NAME=$(echo ${GLOBIMAGE} | cut -d ':' -f1)
+  TAG=$(echo ${GLOBIMAGE} | cut -d ':' -f2)
+
+  STATUSCODE=$(curl -s -o /dev/null -w "%{http_code}" https://${AUTH_CREDS}@${V2_REGISTRY}/v2/${IMAGE_NAME}/manifests/${TAG})
```

Comment:
This will not work in all cases; only if using basic authentication.  This breaks when using DTR or any more advanced method of auth.